src/windowmanagement/data/MenuItems.tsx

```diff
@@ -85,6 +88,7 @@ export const menuItemsData: MenuItem[] = [
     ),
     renderIcon: (size) => <CalendarIcon size={size} />,
     preferredPosition: 'float',
+    floatPosition: { offsetY: 100, offsetX: 370, width: 580, height: 520 },
```

Comment:
Merge in master (to get the polished time input) and make the default size of this panel a bit smaller, please :) Now it seems huge in comparison to the others